pants.toml

```
@@ -137,10 +137,16 @@ pytest_plugins.add = [
 timeout_default = 60
 
 [test]
-# TODO: These are exposed to tests in order to allow for python interpreter discovery when
-# Pants-tests-Pants: in particular, the [python-setup] subsystem consumes them.
-#   see https://github.com/pantsbuild/pants/issues/11638
-extra_env_vars = ["PYENV_ROOT", "HOME", "PATH"]
+extra_env_vars = [
+  # TODO: These are exposed to tests in order to allow for python interpreter discovery when
+  # Pants-tests-Pants: in particular, the [python-setup] subsystem consumes them.
+  #   see https://github.com/pantsbuild/pants/issues/11638
+  "PYENV_ROOT",
+  "HOME",
+  "PATH",
+  # We'd always like complete backtraces in tests.
+  "RUST_BACKTRACE=1",
```

```suggestion
  "RUST_BACKTRACE=full",
```

Not sure which is handier here, I tend to just default to full everywhere

---

Yea... I've never been able to interpret the extra information. Will stick to the simpler rendering for now. Thanks!